.gitignore

```diff
@@ -298,7 +298,7 @@ stage/
 !pal/prebuilt/idl/*_i.c
```

Comment:
`!pal/prebuilt/idl/*_i.c` can be deleted too.

Reply:
Done.